src/core/index.js

```diff
@@ -166,6 +166,11 @@ class IPFS extends EventEmitter {
       this.log('EXPERIMENTAL pubsub is enabled')
     }
     if (this._options.EXPERIMENTAL.ipnsPubsub) {
+      if (!this._options.EXPERIMENTAL.pubsub) {
+        this.log('EXPERIMENTAL pubsub is enabled to use IPNS pubsub')
+        this._options.EXPERIMENTAL.pubsub = true
+      }
```

Comment:
Does go-ipfs do IPNS over PubSub as an experimental flag? Why not just one flag that enables all of PubSub

Reply:
Yes, `go-ipfs` uses an experimental flag for IPNS over Pubsub: `--enable-namesys-pubsub`

Reply:
"Experimental features on Alpha software"